### PR TITLE
Bump spikeinterface version

### PR DESCRIPTION
**Pull request by h-mayorquin**
New spikeinterface release that might help us solve the issues that arose in:

https://github.com/catalystneuro/neuroconv/issues/509

Let's test.

### CODE REVIEW

**Comment by h-mayorquin** [comment]
New release of neo available:
https://pypi.org/project/neo/

File: src/neuroconv/datainterfaces/ecephys/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-spikeinterface>=0.97,<0.98
+spikeinterface>=0.98.1
+neo>=0.12.0
```
Comment (h-mayorquin) [inline src/neuroconv/datainterfaces/ecephys/requirements.txt:2]:
```suggestion
neo<=0.11.1
```

File: src/neuroconv/datainterfaces/ecephys/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
-spikeinterface>=0.97,<0.98
+spikeinterface>=0.98.1
+neo<=0.11.1
```
Comment (bendichter) [inline src/neuroconv/datainterfaces/ecephys/requirements.txt:2]: why are you forcing neo to not use the latest release?

Comment (h-mayorquin) [reply src/neuroconv/datainterfaces/ecephys/requirements.txt:2]: Yes, it should be allowed.

There was a problem with the latest release and spikeinterface extractors that I just fixed:

https://github.com/SpikeInterface/spikeinterface/pull/1848

(That's where the constrain was coming from)

There will be a new release of Spikeinterface with the fix above within this week.

Comment (h-mayorquin) [reply src/neuroconv/datainterfaces/ecephys/requirements.txt]: All right, this is corrected now, this is ready.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin Thanks!

**Pull request merged**